v2/user_obj.go

```diff
@@ -42,22 +42,28 @@ func userFieldStringArray(arr []UserField) []string {
 	return strs
 }
 
+// User entities structure as defined at https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/user
+type UserEntitiesObj struct {
+	URL         *EntitiesObj `json:"url,omitempty"`
```

Comment:
I am not seeing in the URL entity where the annotations, hash tags, mentions, and cash tags are included.

you could have it like this
```go
type UserURLEnttityObj struct {
    URLs        []EntityURLObj        `json:"urls"` 
}
type UserEntitiesObj struct {
    URL *UserURLEnttityObj   `json:"url"`
    Description *EntitiesObj    `json:"description"`
}

```